GNUmakefile

```diff
@@ -2,34 +2,36 @@ AGDA_EXEC ?= agda
 AGDA_OPTIONS=-Werror
 AGDA_RTS_OPTIONS=+RTS -M4.0G -H3.5G -A128M -RTS
 AGDA=$(AGDA_EXEC) $(AGDA_OPTIONS) $(AGDA_RTS_OPTIONS)
+CABAL_EXEC ?= cabal
```

Comment:
Did you mean to call this `CABAL` (because you are referring to it as such below).

Reply:
Oops thanks for catching the bug. I'll fix them later.

Reply:
Ok then I flag this as "changes requested".

Reply:
@andreasabel finished!